requirements.txt

```diff
@@ -30,24 +30,30 @@ mccabe==0.6.1
 mock==2.0.0
 nose==1.3.7
 numpy==1.16.2
+oauthlib==3.0.1
 pbr==5.1.3
 pep8==1.7.1
 Pillow==6.0.0
 psycopg2-binary==2.7.7
 PyJWT==1.4.2
+pynliner==0.8.0
 pyparsing==2.3.1
+pyquery==1.4.0
 python-dateutil==2.8.0
 python-decouple==3.1
 python-dotenv==0.10.1
 python-http-client==3.1.0
+python3-openid==3.1.0
 pytz==2018.9
+readtime==1.1.1
 requests==2.21.0
+requests-oauthlib==1.2.0
+selenium==3.141.0
```

Comment:
selenium is a testing tool. I think it should be in dev requirements

Reply:
Let me move it to the dev-requirements.